platform/android/gradle/dependencies.gradle

```diff
@@ -6,6 +6,12 @@ ext {
             compileSdkVersion: 33,
     ]
 
+    androidSdkVersions = [
```

Comment:
Curious, why would we not merge `androidVersions` and `androidSdkVersions` array together? Semantically, what is the significance of keeping them separate?

Reply:
One is releated to build tools, other to target Android versions. But I can merge them if you want.

Reply:
If there is a semantic difference, I am okay to have them as is. I was just curious.